src/wikisearch.ts

```diff
@@ -96,23 +96,23 @@ export class WikiSearcher {
     }
     console.log("Reading items.");
     const itemFile = await axios(
-      "https://sourceforge.net/p/kolmafia/code/HEAD/tree/src/data/items.txt?format=raw"
+      "https://raw.githubusercontent.com/kolmafia/kolmafia/main/src/data/items.txt"
     );
     for (let line of itemFile.data.split(/\n/)) {
       try {
         const item = new Item(line, itemMap);
         if (item.name()) {
           this._thingMap.set(item.name(), item);
           if (item.get().types.includes("avatar")) {
-            avatarPotionSet.add(item.name())
+            avatarPotionSet.add(item.name());
```

Comment:
heh, I like this one

Reply:
I assume my linter did this because I have no memory of it